modules/tensor_mechanics/test/tests/beam/dynamic/tests

```diff
@@ -1,173 +1,287 @@
+                  " if all the nodal positions provided in the nodal mass file"
+                  " cannot be found in the given boundary or mesh block."
   [../]
   [./error_12]
     type = 'RunException'
     input = 'dyn_euler_small_added_mass_gravity.i'
     cli_args = "NodalKernels/force_y2/nodal_mass_file=nodal_mass.csv NodalKernels/force_y2/mass=2.0 Outputs/csv=false Outputs/exodus=false"
     expect_err = "NodalGravity: Please provide either mass or nodal_mass_file as input, not both."
+
+    requirement = "The NodalGravity class shall produce an error if"
+                  " nodal mass is provided as input both as a constant value"
+                  " and also using a csv file."
   [../]
   [./error_13]
     type = 'RunException'
     input = 'dyn_euler_small_added_mass_gravity.i'
     cli_args = "Outputs/csv=false Outputs/exodus=false"
     expect_err = "NodalGravity: Please provide either mass or nodal_mass_file as input."
+
+    requirement = "The NodalGravity class shall produce an error if"
+                  " nodal mass is not provided as input either as a constant value"
+                  " or using a csv file."
   [../]
   [./error_14]
     type = 'RunException'
     input = 'dyn_euler_small_added_mass_gravity.i'
     cli_args = "NodalKernels/force_y2/nodal_mass_file=nodal_mass_col_test.csv Outputs/csv=false Outputs/exodus=false"
     expect_err = "NodalGravity: The number of columns in"
+
+    requirement = "The NodalGravity class shall produce an error"
+                  " if the number of columns in the nodal mass file is not 4."
   [../]
   [./error_15]
     type = 'RunException'
     input = 'dyn_euler_small_added_mass_gravity.i'
     cli_args = "NodalKernels/force_y2/nodal_mass_file=nodal_mass.csv NodalKernels/force_y2/boundary='right' Outputs/csv=false Outputs/exodus=false"
     expect_err = "NodalGravity: Out of 2 nodal positions in"
+
+    requirement = "The NodalGravity class shall produce an error"
+                  " if all the nodal positions provided in the nodal mass file"
+                  " cannot be found in the given boundary or mesh block."
   [../]
   [./error_16]
     type = 'RunException'
     input = 'dyn_euler_small_rayleigh_hht_action.i'
     cli_args = 'Modules/TensorMechanics/LineElementMaster/all/add_dynamic_variables=false Outputs/csv=false Outputs/exodus=false'
     expect_err = "LineElementAction: When using 'dynamic_consistent_inertia', 'dynamic_nodal_rotational_inertia' or '_dynamic_nodal_translational_inertia', the velocity and acceleration AuxVariables and the corresponding AuxKernels are automatically set by"
+
+    requirement = "The LineElmentAction class shall produce an error if add_dynamic_variables"
+                  " option is set to false while dynamic_consistent_inertia,"
+                  " dynamic_nodal_rotational_inertia or dynamic_nodal_translational_inertia"
+                  " options are set to true."
```

Comment:
Put backticks around parameter names. The system lets you do that and they show up in the typewriter font.